modules/heat_conduction/include/base/HeatConductionNames.h

```diff
@@ -0,0 +1,28 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include <string>
+
+namespace HeatConduction
+{
+static const std::string emissivity = "emissivity";
+static const std::string T_ambient = "T_ambient";
+
+namespace DefaultValues
+{
+static const Real epsilon = 1e-8;
```

Comment:
Haha this is another copy from Pronghorn. This is used in `FVThermalResistanceBC` but we could delete this and use 1e-8 directly in that object or make it a user-settable parameter and then delete this `static`

Reply:
This is used in this context as like a "minimum emissivity in order to care" kind of thing.
```c++
  // iterate to find the approximate surface temperature needed for evaluating the
  // radiation resistance. We only do this iteration if we have radiation transfer.
  if (_emissivity > HeatConduction::DefaultValues::epsilon)
    while (norm > (_tolerance * _alpha))
    {
      T_surface_previous = _T_surface;

      // compute the flux based on the conduction part of the circuit
      flux = (_u[_qp] - _T_surface) / _conduction_resistance;

      computeParallelResistance();

      // use the flux computed from the conduction half to update T_surface
      _T_surface = flux * _parallel_resistance + _T_ambient;
      _T_surface = _alpha * _T_surface + (1 - _alpha) * T_surface_previous;
      norm = std::abs(_T_surface - T_surface_previous) / std::abs(T_surface_previous);

      if (iteration == _max_iterations)
      {
        mooseWarning("Maximum number of iterations reached in 'FVThermalResistanceBC'!");
        break;
      }
      else
        iteration += 1;
    }
```
I think in that context, I'm just going to hard-code 1e-8 in the place of `HeatConduction::DefaultValues::epsilon`. I think it might be confusing to a user to have to supply a iteration tolerance and a minimum emissivity tolerance. If you disagree, I'm open to changing